Components/Badges/badge.css

```diff
@@ -5,55 +5,55 @@
 
 
 .badge-top{
-    width:20px;
-    height:20px;
+    width:2rem;
+    height:2rem;
     border-radius: 50%;
     font-weight:700;
     position: relative;
-    bottom:50px;
-    left:30px;
+    bottom:5rem;
+    left:3rem;
     
 }
 
 
 .badge-bottom-sm{
-    width:20px;
-    height:20px;
+    width:2rem;
+    height:2rem;
     border-radius: 50%;
     font-weight:700;
     position:relative;
-    bottom:20px;
-    left:40px;
+    bottom:2rem;
+    left:4rem;
     
 }
 
 
 .badge-bottom-md{
-    width:20px;
-    height:20px;
+    width:2rem;
+    height:2rem;
     border-radius: 50%;
     font-weight:700;
     position:relative;
-    bottom:25px;
-    left:60px;
+    bottom:2.5rem;
+    left:6rem;
     
 }
 
 
 .badge-bottom-lg{
-    width:20px;
-    height:20px;
+    width:2rem;
+    height:2rem;
     border-radius: 50%;
     font-weight:700;
     position:relative;
-    bottom:25px;
-    left:70px;
+    bottom:2.5rem;
+    left:7rem;
     
 }
 
 
```

Comment:
![image](https://user-images.githubusercontent.com/65125998/152635022-ae92bf08-d91a-4c28-bb59-63609e19691e.png)
1. See you are repeating the code border-radius 50% AND font-weight so you can creat a class 
badge{
    border-radius: 50%;
    font-weight:700;
}
And use it everywhere. This also looks good by the way